### ISSUE TITLE
Page audit

### ISSUE THREAD

**Issue by fedorareis** [CruCentralCoast/crucentralcoast.com#14]
I ran a chrome audit the other day and found that there are a lot of things that should be improved on this page. I would like to see most of the suggested improvements implemented, especially suggestions relating to accessibility.

**Comment by fedorareis** [comment]
I will run a new audit and create improvement tasks once the site has been moved to the new architecture.
